lib/local_system.rb

```diff
@@ -94,7 +94,7 @@ def run_command(*args)
   # the directory where to put the files.
   def retrieve_files(filelist, destination)
     begin
-      LoggedCheetah.run("rsync",  "--chmod=go-rwx", "--files-from=-", "/", destination, :stdout => :capture, :stdin => filelist.join("\n") )
+      LoggedCheetah.run("rsync",  "--chmod=go-rwx", "--files-from=-", "/", destination, stdout: :capture, stdin: filelist.join("\n") )
```

Comment:
Line is too long. [134/100]<br>Unnecessary spacing detected.<br>Space inside parentheses detected.